src/Searchable/DefaultImportSource.php

```diff
@@ -90,9 +90,6 @@ private function newQuery(): Builder
         }, $query);
     }
 
-    /**
-     * @return Builder[]|EloquentCollection
-     */
```

Comment:
What is a reason to remove this doc comment?

Reply:
It's already defined in its interface